content/applications/inventory_and_mrp/inventory/barcode/operations/gs1_usage.rst

```diff
@@ -0,0 +1,240 @@
+=================
+GS1 barcode usage
+=================
+
+.. _barcode/operations/gs1_usage:
+
+.. |AI| replace:: :abbr:`A.I. (Application Identifier)`
+.. |GTIN| replace:: :abbr:`GTIN (Global Trade Item Number)`
+
+Setting up GS1 barcodes in Odoo enables businesses to configure their products and make them
+globally recognized across the supply chain. In Odoo, GS1 barcodes streamline various warehouse
+processes, such as receiving packages, by automating data entry and reducing processing times.
+
+The following sections contain examples of how Odoo uses GS1 barcodes provided by the business to
+identify common warehouse items and automate certain warehouse workflows.
+
+.. important::
+   Odoo **does not** create GS1 barcodes. Businesses must provide a unique Global Trade Item Number
+   (GTIN) from GS1. Then, they can combine their existing GS1 barcodes with product and supply chain
+   information (also provided by GS1) to create barcodes in Odoo.
```

Comment:
@StraubCreative Do you think this makes it clear that GTINs are unique and purchased from GS1?